pkg/enemy/attack.go

```diff
@@ -85,17 +85,18 @@ func (e *Enemy) attack(atk *combat.AttackEvent, evt glog.Event) (float64, bool)
 	//check tags
 	if atk.Info.Durability > 0 {
 		//check for ICD first
-		atk.OnICD = !e.WillApplyEle(atk.Info.ICDTag, atk.Info.ICDGroup, atk.Info.ActorIndex)
+		atk.Info.Durability *= combat.Durability(e.WillApplyEle(atk.Info.ICDTag, atk.Info.ICDGroup, atk.Info.ActorIndex))
```

Comment:
I think `WillApplyEle` should be renamed since it's not returning a `bool` value anymore. Maybe something like `CalcAttackDur` would be better?

Reply:
I don't think i should be making this change here. perhaps a separate issue/pr